### PR TITLE
License text in LICENSE file

### PR DESCRIPTION
**Pull request by pchaigno**
This should help various automated tools such as [licensee](https://github.com/benbalter/licensee) recognize the license. Licensee is used, among other things, for the GitHub API.

### CODE REVIEW

**Comment by pchaigno** [comment]
@infininight Did you get a chance to look into this?

**Comment by infininight** [comment]
I've been looking at the benefits of doing this as it's not something I want to do on a piecemeal basis, if adding a LICENSE file I'd do it for all relavant bundles which is a bit of a task. I'm not completely against doing so but I'm struggling to see the benefits. Consulting a license is something that should be done intentionally, I'm not sure how much time an automated system saves. This repository highlights one of my concerns, I consider the addendum after the license to be an important part of the license. Leaving it out is not acceptable but putting it in removes any real automation in checking the license as it will be non-standard.

In short I need some convincing as to the benefits of adding this. With the fact that choosing to use code in another manner is a complicated choice checking the license seems like such a small part of that.

**Comment by pchaigno** [comment]
> Consulting a license is something that should be done intentionally, I'm not sure how much time an automated system saves.

I'm actually trying to push this change in several repositories to improve the process used at [github/linguist](https://github.com/github/linguist) for PR validation. Having the license in its own file would help for the coarse detection of open-source licences during PR reviews (when adding submodules). Of course, a manual verification is still needed when the license can't be automatically recognized.

> This repository highlights one of my concerns, I consider the addendum after the license to be an important part of the license. Leaving it out is not acceptable but putting it in removes any real automation in checking the license as it will be non-standard.

I agree that the addendum is essential. Its inclusion in the license makes it non-standard but not undetectable. If the license has its own file, it would be possible to identify it uniquely (via its hash value) in order to whitelist it. This would take all its meaning with licenses such as the present one, which are shared among a considerable number of repositories. And, if it can be uniquely identified, any modification to the text of the license can also be detected.

Licensee isn't capable to uniquely identify licenses but I'm hoping that it will change (see benbalter/licensee#67).

> I've been looking at the benefits of doing this as it's not something I want to do on a piecemeal basis, if adding a LICENSE file I'd do it for all relavant bundles which is a bit of a task.

If you decide to go down this road, I could probably write a script to push the modification (or to open the PRs) since it's the same for all repositories.

**Comment by pchaigno** [comment]
> Licensee isn't capable to uniquely identify licenses but I'm hoping that it will change (see benbalter/licensee#67).

Licensee is now capable of uniquely identifying licenses, thanks to benbalter/licensee#76.
@infininight It would really help to automate part of the license verification process at [github/linguist](https://github.com/github/linguist) if this pull request was merged ;)

**Comment by Alhadis** [comment]
> In short I need some convincing as to the benefits of adding this.

Currently, this grammar is [causing build failures](https://github.com/github/linguist/pull/3337) at Linguist. All specs are failing, blocking pull requests from being merged and [even confusing contributors](https://github.com/github/linguist/pull/3333#issuecomment-261343563).

I trust that's enough to convey the benefits of merging this.

**Comment by infininight** [comment]
@Alhadis Looking at the linked issues it appears the issue is that linguist now sources the license information in readme files now and the non-standard license of this bundle vs the other TextMate bundles was tripping it up. Since Licensee now can parse license information from the readme this issue appears to be resolved.

**Comment by Alhadis** [comment]
@pchaigno If this happens again, we might be better off simply replacing this grammar with another one.
